pkg/kudoctl/kudoinit/options.go

```diff
@@ -10,18 +10,30 @@ import (
 type Options struct {
 	// Version is the version of the manager `0.5.0` for example (must NOT include the `v` in `v0.5.0`)
 	Version string
+	// Name is used as controller-manager Name and as a prefix for the service. Currently hard-coded
+	// to "kudo-controller-manager" as we don't support setting it in the CLI.
+	Name string
```

Comment:
I would prefer `ControllerName` or `ManagerName` but I'd be ok with this.

Reply:
`ManagerName` it is!